src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -336,6 +333,16 @@ static void CheckForExperimentStatus(ITextView textView, Document document)
                 }
             }
 
+            // We need to remember the trigger location for when a completion service claims expanded items are available
+            // since the intial trigger we are able to get from IAsyncCompletionSession might not be the same (e.g. in projection scenarios,)
```

Comment:
nit:
```suggestion
            // since the initial trigger we are able to get from IAsyncCompletionSession might not be the same (e.g. in projection scenarios)
```